package.json

```diff
@@ -116,6 +116,10 @@
                 "title": "Julia: Lint Package"
             },
             {
+                "command": "language-julia.reload-modules",
+                "title": "Julia: Reload imported modules"
+            },
+            {
```

Comment:
I think this command should be added to the ``activationEvents`` section in package.json.